tests/kafkatest/tests/core/upgrade_test.py

```diff
@@ -94,6 +94,12 @@ def perform_upgrade(self, from_kafka_version, to_message_format_version=None):
             self.wait_until_rejoin()
 
     @cluster(num_nodes=6)
+    @parametrize(from_kafka_version=str(LATEST_3_5), to_message_format_version=None, compression_types=["none"])
+    @parametrize(from_kafka_version=str(LATEST_3_5), to_message_format_version=None, compression_types=["lz4"])
+    @parametrize(from_kafka_version=str(LATEST_3_5), to_message_format_version=None, compression_types=["snappy"])
```

Comment:
A general question because I have been going into this part of the codebase - is there a reason why the other two compression types are not tested (i.e. gzip, zstd)?

Reply:
To be honest I don't know. At this point, we've copied the same lines as the previous releases. Maybe this could be improved. If so I'd rather separate it from this PR and handle this out of a release.

Reply:
Oh, I agree, this definitely needs to not be in this pull request, I was just wondering